plugins/openconfig.py

```diff
@@ -577,3 +594,15 @@ def v_preinit_module_checks(ctx, statement):
         pos = error.Position(statement.pos.ref)
         pos.line = ln_count
         err_add(ctx.errors, pos, 'OC_KEY_ARGUMENT_UNQUOTED', arg_part)
+
+def v_chk_import(ctx, statement):
+  """
+    Validation function run against import statements.
+  """
+  arg = getattr(statement, "arg", None)
+  if arg is not None and arg in COMMON_IMPORT_PREFIXES:
+    prefix = statement.search_one('prefix')
+    if prefix is not None and not prefix.arg == COMMON_IMPORT_PREFIXES[arg]:
```

Comment:
didn't we also want to have a check to make sure that the prefix is of the form `oc-*` ?  Maybe I'm missing it, but I only see the check for the common module prefixes defined at the top.